doc/source/HOW-TO/cluster-configuration.rst

```diff
@@ -1,38 +1,72 @@
-******************************
-How to configure Qserv cluster
-******************************
+*******************************************
+How to configure a multi-node Qserv cluster
+*******************************************
 
-Pre-requisite
-=============
+Pre-requisites
+==============
 
-Build/install Qserv on each machine of the cluster, using eups.
+Build/install Qserv and its pre-requisites on all nodes.
 
-Install master and worker nodes
-===============================
+Configure master and worker nodes
+=================================
 
```

Comment:
add a note somewhere at the beginning that the examples here are for 2-worker cluster.